logicalplan/distribute_avg.go

```diff
@@ -27,10 +27,14 @@ func (r DistributeAvgOptimizer) Optimize(plan parser.Expr, _ *query.Options) par
 			count := *(*current).(*parser.AggregateExpr)
 			count.Op = parser.COUNT
 			*current = &parser.BinaryExpr{
-				Op:             parser.DIV,
-				LHS:            &sum,
-				RHS:            &count,
-				VectorMatching: &parser.VectorMatching{},
+				Op:  parser.DIV,
+				LHS: &sum,
+				RHS: &count,
+				VectorMatching: &parser.VectorMatching{
+					Include:        aggr.Grouping,
```

Comment:
We need to include that to make sure that __name__ is not dropped ( its usually dropped by vector binary "/" )